scripts/tests

```diff
@@ -388,20 +378,44 @@ def kat(force_qemu, verbose, cflags, arch_flags):
     type=bool,
     help="Benchmarking binary is run with sudo.",
 )
+@click.option(
+    "-w",
+    "--exec-wrapper",
+    help="Run the benchmark binary with the user-customized wrapper.",
+)
 @click.option(
     "-t",
     "--mac-taskpolicy",
     nargs=1,
     type=click.Choice(["utility", "background", "maintenance"]),
+    hidden=platform.system() != "Darwin",
     show_default=True,
     default=None,
-    help="Run the program using the specified QoS clamp. Applies to MacOS only. Setting this flag to 'background' guarantees running on E-cores.",
+    help="Run the program using the specified QoS clamp. Applies to MacOS only. Setting this flag to 'background' guarantees running on E-cores. This is an abbreviation of --exec-wrapper 'taskpolicy -c {mac_taskpolicy}'.",
 )
 def bench(
-    force_qemu, verbose, cycles, cflags, arch_flags, output, run_as_root, mac_taskpolicy
+    force_qemu,
+    verbose,
+    cycles,
+    cflags,
+    arch_flags,
+    output,
+    run_as_root,
+    exec_wrapper,
+    mac_taskpolicy,
 ):
     config_logger(verbose)
 
+    if mac_taskpolicy:
+        if exec_wrapper:
+            logging.error(f"cannot set both --mac-taskpolicy and --exec-wrapper")
+            sys.exit(1)
+        else:
+            exec_wrapper = f"taskpolicy -c {mac_taskpolicy}"
+
+    if exec_wrapper:
+        exec_wrapper = exec_wrapper.split(" ")
```

Comment:
I'm thinking, probably it's cleaner to move this into `base_run` function, near L87

Reply:
indeed. I moved it there.